File: Frontend Parcial/mymedicapp/lib/cama.dart
class Cama {
  int? idCama;
  bool? estado;
  int? idPiso;
  String? idPaciente;
  String? nombre;
  String? fechaIngreso;
  String? diagnostico;

  Cama({
    this.idCama,
    this.estado,
    this.idPiso,
    this.idPaciente,
    this.nombre,
    this.fechaIngreso,
    this.diagnostico,
  });

  factory Cama.fromJson(Map<String, dynamic> json) => Cama(
        idCama: json["idCama"],
        estado: json["estado"],
        idPiso: json["idPiso"],
        idPaciente: json["idPaciente"],
        nombre: json["nombre"],
        fechaIngreso: json["fechaHoraIngreso"],
        diagnostico: json["diagnostico"],
      );
}

File: Parcial/Models/Cama.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Parcial.Models
{
    public class Cama
    {
        // DATOS DE LA CAMA
        [Key]
        public int IdCama { get; set; }
        [Required]
        public int Estado { get; set; }
        [Required]
        public int IdPiso { get; set; }

        // DATOS DEL PACIENTE
        [Required(AllowEmptyStrings = true)]
        public String IdPaciente { get; set; }
        [Required(AllowEmptyStrings = true)]
        public String Nombre { get; set; }
        [Required(AllowEmptyStrings = true)]
        public String FechaHoraIngreso { get; set; }
        [Required(AllowEmptyStrings = true)]
        public String Diagnostico { get; set; }
    }
}
